# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-ia64-printk-cont.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.8/arch/ia64/kernel/time.c.vanilla	2024-04-18 13:44:17.633984660 +0200
+++ linux-6.8/arch/ia64/kernel/time.c	2024-04-18 13:47:05.103983925 +0200
@@ -307,10 +307,10 @@
 
 	if (platform_base_drift != -1) {
 		itc_drift = platform_base_drift*itc_ratio.num/itc_ratio.den;
-		printk("+/-%ldppm\n", itc_drift);
+		printk(KERN_CONT "+/-%ldppm\n", itc_drift);
 	} else {
 		itc_drift = -1;
-		printk("\n");
+		printk(KERN_CONT "\n");
 	}
 
 	local_cpu_data->proc_freq = (platform_base_freq*proc_ratio.num)/proc_ratio.den;
